x-pack/platform/plugins/shared/alerting/server/application/rule/methods/mute_alert/mute_instance.ts

```diff
@@ -89,5 +92,14 @@ async function muteInstanceWithOCC(
         updatedAt: new Date().toISOString(),
       }),
     });
+
+    if (indices && indices.length > 0) {
+      await context.alertsService?.muteAlertInstance({
+        ruleId,
+        alertInstanceId,
+        indices,
```

Comment:
I am wondering, do we actually need this? Could the alert service figure it out automatically by the rule ID? @ersin-erdal Wdyt?